libquickevent/libquickeventgui/src/reportoptionsdialog.h

```diff
@@ -40,11 +41,13 @@ class QUICKEVENTGUI_DECL_EXPORT ReportOptionsDialog : public QDialog, public qf:
 	QF_PROPERTY_BOOL_IMPL2(p, P, ageLayoutVisible, false)
 	QF_PROPERTY_BOOL_IMPL2(c, C, olumnCountEnable, true)
 	QF_PROPERTY_BOOL_IMPL2(r, R, esultOptionsVisible, false)
+	QF_PROPERTY_BOOL_IMPL2(s, S, tartTimesTypesVisible, false)
 private:
 	using Super = QDialog;
 public:
 	enum class BreakType : int {None = 0, Column, Page};
 	enum class FilterType : int {WildCard = 0, RegExp, ClassName};
+	enum class StartTimesType : int {FromZero = 0, DayTime};
```

Comment:
```c++
enum class StartTimeFormat {RelativeToClassStart = 0, DayTime};
```

Reply:
Prijde mi to zbytecne, ale udelam to.

Reply:
Zbytecny ti to pripadat muze, ja verim, ze ostatni oceni, ze se enum nejmenuje nejasne `FromZero`, ale skutecne semanticky podle toho, co vyjadruje